mlabs/efficient-nft-pab/Main.hs

```diff
@@ -184,26 +184,40 @@ main = do
              , mp'fakeAuthor = Just auth
              , mp'feeVaultKeys = []
              }
+
+  Hask.putStrLn $ "currencySymbol: " <> Hask.show uCs
+
   Hask.putStr "seabug-mint-request: "
   ByteString.putStrLn
     $ JSON.encode
     $ Mint (assetClass uCs $ fromString tokenName, mp)
 
   let c = NftCollection
         { nftCollection'collectionNftCs = uCs
-        , nftCollection'lockLockup = 1
-        , nftCollection'lockLockupEnd = 1
         , nftCollection'lockingScript = validatorHash $ lockValidator uCs 1 1
         , nftCollection'author = auth
         , nftCollection'authorShare = toEnum 1000
         , nftCollection'daoScript = validatorHash $ daoValidator []
         , nftCollection'daoShare = toEnum 500
+        -- unused:
+        , nftCollection'lockLockup = 1
+        , nftCollection'lockLockupEnd = 1
         }
 
+  Hask.putStr "unapplied-minting-policy: "
+  ByteString.putStrLn $ JSON.encode Token.policyDataScript
+
+  Hask.putStr "unapplied-minting-policy hash: "
+  Hask.print $ scriptHash Token.unappliedPolicyScript
```

Comment:
Some of these logs here can go now, since we've verified it works.

Reply:
Removed useless output.